### DISCUSSION TITLE
How to Use Fastlane Snapshot with Rosetta for iOS Apps?

### DISCUSSION THREAD

**Discussion by HaydenCampbell** [fastlane/fastlane#21715] [Q&A]
I am developing an iOS app that requires localized screenshots for different devices. I use fastlane snapshot to automate this process, but I encounter a problem: my app uses Google ML Kit, which is not compatible with arm64 simulators. Therefore, I need to run fastlane snapshot with Rosetta build and Rosetta Simulators.

How can I achieve this? I would appreciate any advice or tips.

Thank you for your time!

**Comment by rttss-sahil** [comment]
Same issue. Were you able to solve this?
